package.json

```diff
@@ -47,6 +47,7 @@
   "dependencies": {
     "bn": "^1.0.1",
     "ethereumjs-util": "^5.1.1",
+    "flexboxgrid": "^6.3.1",
```

Comment:
Your right, i think maybe the best option would be using react as a dependency but what do you think?